### PR TITLE
Get System Language - LN

### PR DESCRIPTION
**Pull request by e1e5en-gd**
Get System Language

get_system_language

Category: _Native_
To get the current system language according to the IS0 639 standard.
The _Krom_ platform always returns “_en_”.

### CODE REVIEW

File: blender/arm/logicnode/native/LN_get_system_language.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from arm.logicnode.arm_nodes import *
+
+class GetSystemLanguage(ArmLogicTreeNode):
+    """Returns the name of the current system."""
```
Comment (MoritzBrueckner) [inline blender/arm/logicnode/native/LN_get_system_language.py:4]: I think the docstring is left over from another node?

Comment (e1e5en-gd) [reply blender/arm/logicnode/native/LN_get_system_language.py:4]: Yes. Sorry

**Comment by luboslenco** [comment]
> The Krom platform always returns “en”.

Which OS are you on? It should work everywhere apart from Linux - that one is still missing here:
https://github.com/Kode/Kinc/blob/master/Backends/System/Linux/Sources/Kore/System.cpp#L846

**Comment by e1e5en-gd** [comment]
I have Windows 10 and language ru.
Krom gave out en, so I looked at its sources (https://github.com/Kode/Kha/blob/5a489c1305b219e2ad1e9d089c32d8a53a21d4f7/Backends/Krom/kha/SystemImpl.hx#L223). I did not look into the source code of Linux.

**Pull request merged**